### ISSUE TITLE
Magenta on window?

### ISSUE THREAD

**Issue by rupeshs** [tensorflow/magenta#429]
Happy that tensor flow for windows released but what about magenta will it work  on windows?

**Comment by Banus** [comment]
Currently Tensorflow for Windows is available only for Python 3.5, and Python 2.7 is not supported even when compiling from source (see e.g. [cmake instructions](https://github.com/tensorflow/tensorflow/tree/r0.12/tensorflow/contrib/cmake)).
Thus, Windows support relies critically on the support for Python3 (see PR #461).

**Comment by arijit17** [comment]
When can we expect Magenta to work on Windows 10?

**Comment by jsleep** [comment]
@arijit17 That would be once all of Magenta's source code has been translated to Python 3, which is probably not at the top of the priority list.

**Comment by gbernal** [comment]
Ah.. I came here looking for the answer to this questions.. Is everyone running some form of Linux?

**Comment by icoxfog417** [comment]
You can use [bash on Windows](https://msdn.microsoft.com/ja-jp/commandline/wsl/about). My platform is Windows 10 and executes the Magenta by bash on Windows. 
I hope the short instruction on my repository helps you.

[icoxfog417/magenta_session#install](https://github.com/icoxfog417/magenta_session#install)

**Comment by gbernal** [comment]
thanks for the advice. I'm still having some issues. I'm now having this issue https://stackoverflow.com/questions/42013316/after-building-tensorflow-from-source-seeing-libcudart-so-and-libcudnn-errors 
Do you install windows cuDNN  or linux cuDNN?

**Comment by icoxfog417** [comment]
@gbernal Now [bash on Windows is not supported GPU](https://github.com/Microsoft/BashOnWindows/issues/829).  We have to wait Magenta supports Python3 or bash on Windows supports GPU!

**Comment by AlexandreJup** [comment]
Hello ! 
Any news on running magenta in windows? (not using bash on Windows)

**Comment by dubreuia** [comment]
@AlexandreJup I'm using Magenta in Windows (with cmd, and PyCharm), no problem for now, so it seems the support for Python 3.x has been fixed. I haven't tested all the models and training for now, but it looks OK. I'll report back if I see anything that doesn't work.

Some things to know:

- You will need Python 3.5.x (I'm using 3.5.6), because `tensorflow-gpu` requires it (not sure if necessary if you only need `tensorflow`)
- You will need CUDA 10.0 (not 10.1), same reason
- I'm using CudRNN 7.6 (not sure if there is a specific version to have), same reason

**Comment by AlexandreJup** [comment]
@dubreuia Awesome, I will have a look at it then. You confirm that you are running tensorflow-gpu? That was the reason of my question in the first place, I am trying to run tensorflow-gpu but was having issues.

**Comment by dubreuia** [comment]
@AlexandreJup Yes `magenta_gpu-1.1.2` and `tensorflow_gpu-1.14.0`. Check my previous comment for the specific versions you need.

**Comment by AlexandreJup** [comment]
Thank you, I will try with your versions !
